cli/main.rs

```diff
@@ -701,6 +701,21 @@ async fn run_command(
     return run_from_stdin(flags).await;
   }
 
+  if flags.to_permission_args().is_empty()
+    && flags.argv.iter().any(|arg| {
+      arg.contains("--allow-env")
+        || arg.contains("--allow-hrtime")
+        || arg.contains("--allow-net")
+        || arg.contains("--allow-read")
+        || arg.contains("--allow-run")
+        || arg.contains("--allow-sys")
+        || arg.contains("--allow-write")
+        || arg.contains("--allow-all")
```

Comment:
We could move this into a method on `Flags` itself because that will allow us to keep the knowledge of what the text of the arguments are (ex. `--allow-read`, etc.) buried within the flags.rs module (ex. `flags.has_permission()` and `flags.has_permission_in_argv()` or something like that).. It will also help keep the code of close beside `to_permission_args` and then we could write a quick unit test for this method.

Reply:
I usually prefer a functional approach. Is there any particular advantage over having a method instead of a simple function, or will a simple function suffice?

Reply:
One way is `has_permission(&flags)` the other is `flags.has_permission()`. The second way is much more discoverable.